Tasks/ipa-resign/task.json

```diff
@@ -12,8 +12,8 @@
     "demands": [ "xcode" ],
     "version": {
         "Major": "1",
-        "Minor": "178",
-        "Patch": "2"
+        "Minor": "182",
```

Comment:
Please bump version to current sprint number

Reply:
Thanks! Resolved.